modules/gdnative/doc_classes/GDNativeLibrary.xml

```diff
@@ -1,35 +1,50 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GDNativeLibrary" inherits="Resource" category="Core" version="3.2">
 	<brief_description>
+		An external library containing functions or script classes to use in Godot.
 	</brief_description>
 	<description>
+		A GDNative library can implement [NativeScript]s, global functions to call with the [GDNative] class, or low-level engine extensions through interfaces such as [ARVRInterfaceGDNative]. The library must be compiled for each platform and architecture that the project will run on.
 	</description>
 	<tutorials>
+		<link>https://docs.godotengine.org/en/latest/tutorials/plugins/gdnative/gdnative-c-example.html</link>
+		<link>https://docs.godotengine.org/en/latest/tutorials/plugins/gdnative/gdnative-cpp-example.html</link>
 	</tutorials>
 	<methods>
 		<method name="get_current_dependencies" qualifiers="const">
 			<return type="PoolStringArray">
 			</return>
 			<description>
+				Returns paths to all dependency libraries for the current platform and architecture.
 			</description>
 		</method>
 		<method name="get_current_library_path" qualifiers="const">
 			<return type="String">
 			</return>
 			<description>
+				Returns the path to the dynamic library file for the current platform and architecture.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="config_file" type="ConfigFile" setter="set_config_file" getter="get_config_file">
+			This resource in INI-style [ConfigFile] format, as in [code].gdnlib[/code] files.
 		</member>
 		<member name="load_once" type="bool" setter="set_load_once" getter="should_load_once" default="true">
+			If [code]true[/code] Godot loads only one copy of the library and each script that references the library will share static data like static or global variables.
```

Comment:
Style-wise, this should be `If [code]true[/code], ` (i.e. with comma after true/false) to be consistent with the rest of the docs.

Reply:
Ah, I was wondering about this - the writing guidelines page still has [an old example without the comma](https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-if-true-to-describe-booleans)